README.md

```diff
@@ -21,7 +21,7 @@
 [![version][version-badge]][package]
 [![MIT License][license-badge]][LICENSE]
 
-[![All Contributors](https://img.shields.io/badge/all_contributors-20-orange.svg?style=flat-square)](#contributors)
+[![All Contributors](https://img.shields.io/badge/all_contributors-21-orange.svg?style=flat-square)](#contributors)
```

Comment:
It seems that your user info has not been added to the readme.

Reply:
Looks like a manual change. Otherwise the table would have been updated accordingly.

Reply:
Thank you. I followed Berkmann18's PR and changed it manually. I'll fix it.

Reply:
@Jongjineee If you want to add yourself as a contributor, make sure you do it through the [`all-contributors-cli`](https://www.npmjs.com/package/all-contributors-cli) package.
In any case, the bot or us can do that.

Reply:
@Berkmann18 I want you to do it. So there's no problem with the code?